users/serializers.py

```diff
@@ -1,11 +1,16 @@
 from django.contrib.auth.password_validation import validate_password
+from drf_extra_fields.fields import Base64ImageField
 from rest_framework import serializers
 
 from users.models import User
 
 
 class UserPublicSerializer(serializers.ModelSerializer):
     id = serializers.IntegerField(read_only=True)
+    profile_photo = Base64ImageField(
+        max_length=None,
```

Comment:
Not sure we need to set it explicitly as it is `None` by default
```
# rest_framework.fields

class FileField(Field):
...
def __init__(self, **kwargs):
    ...
    self.max_length = kwargs.pop('max_length', None)
...
```

Reply:
Removed